GNUmakefile

```diff
@@ -81,13 +77,6 @@ tools:  ## Install dev tools
 	go install github.com/hashicorp/terraform-plugin-codegen-framework/cmd/tfplugingen-framework@latest
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin $(GOLANGCI_VERSION)
 
-.PHONY: check
-check: test lint
```

Comment:
why do we remove this? I would include additional check to this command such as fmtcheck

Reply:
it's not being used in GH actions and i don't think anybody is using this

Reply:
ah okay, I was probably the only one using it xD happy to remove it then